IntegrationTests/Package.swift

```diff
@@ -146,6 +146,7 @@ func addIntegrationTestTarget(_ name: String) {
 
 let servicesWithIntegrationTests: [String] = [
     "AWSCloudFrontKeyValueStore",
+    "AWSDynamoDB",
```

Comment:
DynamoDB is used to verify that the client handles Account ID and endpoint mode correctly.